tests/installation/scc_registration.pm

```diff
@@ -24,7 +24,7 @@ sub run {
         assert_registration_screen_present();
         fill_in_registration_data();
         wait_still_screen();
-        if (is_sle('>=15-SP4') && check_screen('import-untrusted-gpg-key', 5)) {
+        if ((is_sle('>=15-SP4') || is_sle_micro) && check_screen('import-untrusted-gpg-key', 5)) {
```

Comment:
for SLE we handling keys only for latest unreleased version , which means if such issue will suddenly pop-up somewhere like 15 SP2 this will be treated as a bug and fail the test . Are you sure that you want blindly accept unknown gpg keys on SLEM5.1/5.2/5.3 ?

Reply:
Good point. I'm planning to run this in "latest", so I'll add the version check here. Thanks!